### ISSUE TITLE
Error message navigating to the editor tab of another file.

### ISSUE THREAD

**Issue by sound-fx** [spyder-ide/spyder#5217]
## Description

**What steps will reproduce the problem?**

I can't yet reproduce the problem reliably.  Here are the steps for this one instance:

1. Type Ctrl-P to pop up the list of open files.
2. Type the first one or two characters of a file name from the list.
3. I don't remember whether I used the mouse to click on the desired file, or used the up/down arrow keys.
4. Press the Enter key.

**What is the expected output? What do you see instead?**
I expected the editor tab for my desired file to come to the top.  This error message appeared in addition to the editor tab rising to the top.

**Please provide any additional information below**

  File "C:\Users\Greg\AppData\Local\Programs\Anaconda2-64\lib\site-packages\spyder\widgets\editortools.py", line 490, in clicked
    self.activated(item)
  File "C:\Users\Greg\AppData\Local\Programs\Anaconda2-64\lib\site-packages\spyder\widgets\editortools.py", line 481, in activated
    if _id == editor_id and editor.parent() is parent:
RuntimeError: wrapped C/C++ object of type CodeEditor has been deleted

## Version and main components

* Spyder Version: 3.2.2 
* Python Version: 2.7.13
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.5.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.20.3 (OK)
numpy >=1.7       :  1.13.1 (OK)
sphinx >=0.6.6    :  1.6.3 (OK)
rope >=0.9.4      :  0.9.4 (OK)
jedi >=0.9.0      :  0.10.2 (OK)
nbconvert >=4.0   :  5.1.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.26 (OK)
qtconsole >=4.2.0 :  4.3.0 (OK)
IPython >=4.0;<6.0:  5.3.0 (OK)
pylint >=0.25     :  1.7.2 (OK)

```

**Comment by jnsebgosselin** [comment]
@ccordoba12 Possible duplicate of Issue #3390

@sound-fx thanks for your bug report. I think this has been fixed in 3.2.3. Could you please update your Spyder version and see if the bug still occur? Be aware though that this Issue #5057 has not been resolved yet. So Ctrl-P fast file switching is not working yet in split editor panel, only on the main one.

**Comment by ccordoba12** [comment]
@sound-fx, I'm closing this one for now. Please reopen it in case this problem is not fixed for you in Spyder **3.2.3**.
